components/modal/Overlay.js

```diff
@@ -14,7 +15,7 @@ const Overlay = ({ isClosing = false, className: extraClassName = '', onExit, ..
         }
     };
 
-    const className = [CLASSES.OVERLAY, isClosing && CLASSES.OVERLAY_OUT, extraClassName].filter(Boolean).join(' ');
```

Comment:
Because the helper `classnames` is taking care of the filtering and joining, maybe we can avoid defining `className` and write directly:
```
    return (
        <Portal>
            <div
                className={classnames([CLASSES.OVERLAY, isClosing && CLASSES.OVERLAY_OUT, extraClassName])}
                onAnimationEnd={handleAnimationEnd}
                {...rest}
            />
        </Portal>
    );
```
We do that in most of the other components, so the code would be again more homogeneous ;)